middlewares/validators/user.js

```diff
@@ -189,6 +189,10 @@ async function validateUserQueryParams(req, res, next) {
         .optional(),
       role: joi.string().valid(ROLES.MEMBER, ROLES.INDISCORD, ROLES.ARCHIVED).optional(),
       verified: joi.string().optional(),
+      time: joi
+        .string()
+        .regex(/^[1-9]\d*d$/)
+        .optional(),
```

Comment:
instead of `time` can we call `since`. it suits better as per me. ignore my comment if other are ok with `time`